common/src/main/scala/explore/components/undo/UndoRegion.scala

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components.undo
+
+import react.common.ReactProps
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import cats.effect.IO
+import cats.implicits._
+import monocle.macros.Lenses
+import crystal.react.implicits._
+import cats.effect.Async
+import cats.kernel.Monoid
+
+final case class UndoRegion[M](
```

Comment:
IS this is a component that goes around the component where you want to undo?

Reply:
Yes, it's what was called `Undoer` before. But now `Undoer` is a trait independent of React, and `UndoRegion` implements it.